src/ctree-app/ctree-app.html

```diff
@@ -212,21 +222,25 @@
       <app-drawer slot="drawer" id="navDrawer" class="nav-drawer" swipe-open>
         <app-toolbar>cTrees</app-toolbar>
         <div class="nav-content">
-          <paper-input label="Search" no-label-float>
+          <paper-input label="Search" value="{{searchText}}" no-label-float>
             <iron-icon icon="search" slot="prefix"></iron-icon>
+            <paper-icon-button slot="suffix" on-tap="clearSearch" icon="clear" alt="clear" title="clear"></paper-icon-button>
           </paper-input>
 
           <iron-selector selected="[[page]]" attr-for-selected="name" class="drawer-list" fallback-selection="all" role="navigation">
-            <iron-list items="[[navList]]" as="item">
-              <template>
-                <a name="[[item.name]]" href="[[rootPath]]/[[item.name]]">
+            <template is="dom-repeat" items="[[getCTreeList(navList.splices,rootPath,searchText)]]" as="item">
+              <div>
+                <a name="[[item.name]]" class="ctree-item" href="[[rootPath]]/[[item.name]]">
                   <paper-button>
                     <iron-icon src="[[item.data.icon]]"></iron-icon>
                     [[item.data.label]]
                   </paper-button>
                 </a>
-              </template>
-            </iron-list>
+                <template is="dom-if" if="[[searchText]]">
+                  <paper-toggle-button class="green" checked="{{item.data.saved}}" title="save"></paper-toggle-button>
```

Comment:
I'm not seeing this yet.  It should appear for all items when `searchText` is not empty.